docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst

```diff
@@ -5,56 +5,70 @@ How to validate data without a Checkpoint
+        .. code-block:: python
 
-.. code-block:: python
+            # If you would like to validate an entire table or view in your database's default schema:
+            batch_kwargs = {'table': "YOUR_TABLE", 'datasource': datasource_name}
 
-    # If you would like to validate a file on a filesystem:
-    batch_kwargs = {'path': "YOUR_FILE_PATH", 'datasource': datasource_name}
+            # If you would like to validate an entire table or view from a non-default schema in your database:
+            batch_kwargs = {'table': "YOUR_TABLE", "schema": "YOUR_SCHEMA", 'datasource': datasource_name}
 
-    # If you would like to validate in a PySpark or Pandas dataframe:
-    batch_kwargs = {'dataset': "YOUR_DATAFRAME", 'datasource': datasource_name}
+            # If you would like to validate the result set of a query:
+            batch_kwargs = {'query': 'SELECT YOUR_ROWS FROM YOUR_TABLE', 'datasource': datasource_name}
 
-Finally, we create the batch using those ``batch_kwargs`` and the name of the Expectation Suite we want to use, and run validation:
+        The following ``batch_kwargs`` can be used to create a batch for a Pandas or PySpark Datasource:
 
-.. code-block:: python
+        .. code-block:: python
 
-    batch = context.get_batch(batch_kwargs, "my_expectation_suite")
+            # If you would like to validate a file on a filesystem:
+            batch_kwargs = {'path': "YOUR_FILE_PATH", 'datasource': datasource_name}
 
-    results = context.run_validation_operator(
-        "action_list_operator",
-        assets_to_validate=[batch],
-        run_id="my_run_id") # Make my_run_id a unique identifier, e.g. a timestamp
+            # If you would like to validate in a PySpark or Pandas dataframe:
+            batch_kwargs = {'dataset': "YOUR_DATAFRAME", 'datasource': datasource_name}
+
+        Finally, we create the batch using those ``batch_kwargs`` and the name of the Expectation Suite we want to use, and run validation:
+
+        .. code-block:: python
+
+            batch = context.get_batch(batch_kwargs, "my_expectation_suite")
+
+            results = context.run_validation_operator(
+                "action_list_operator",
+                assets_to_validate=[batch],
+                run_id="my_run_id") # Make my_run_id a unique identifier, e.g. a timestamp
+
+
+        This runs validation and executes any ValidationActions configured for this ValidationOperator (e.g. saving the results to a ValidationResult Store).
+
+    .. tab-container:: tab1
+        :title: Show Docs for Experimental API (0.13)
+
+        .. Attention:: As part of the new modular expectations API in Great Expectations, Validation Operators are evolving into Checkpoints. This means running a Validation without a Checkpoint is no longer supported in Great Expectations version 0.13 or later. Please read :ref:`Checkpoints and Actions<checkpoints_and_actions>` to learn more.
```

Comment:
That's version 0.13.8 and later. See the other Checkpoints docs for the tabs labeling and terminology (it's different... sorry._

Reply:
Thanks for catching this. I'll update now :)